pkg/property/unmarshal_test.go

```diff
@@ -34,6 +35,9 @@ func TestUnmarhsalPropertiesThreadThroughRegisterResource(t *testing.T) {
 	type testCase struct {
 		name   string
 		inputs resource.PropertyMap
+
+		// Only needed if turnaround value is a normalized form of the original value.
```

Comment:
Is "turnaround value" an official term? 
Under what conditions does normalization happen?

Reply:
NO it's not. I've reworded this. See `unknown`, `secret` and `output-known` cases for the conditions where normalization happens currently.